GitUI/ScriptsEngine/IRunScript.cs

```diff
@@ -4,5 +4,14 @@
     public interface IRunScript
     {
         void Execute(int scriptId);
+
+        void Execute(int scriptId, IFileListSource fileListSource);
+
+        public interface IFileListSource
+        {
+            IEnumerable<string>? GetFiles();
+
+            int? GetLineNumber();
```

Comment:
Can you please describe the use-case for line number? And how does this API works with `GetFiles()`?

Reply:
One use case is to open a user-chosen editor at the code line of the diff. Several times, I noticed that this is a missing feature.

Reply:
#11191 should have been implemented a long time ago and it is what I miss the most with the VS integration.